### ISSUE TITLE
What learning rate is preferable for very close by pixel numbers of ground truth and background ?

### ISSUE THREAD

**Issue by EldorIbragimov** [jwyang/faster-rcnn.pytorch#76]
Also, what learning steps are good for my case. Please share your experience. Thanks for help!

**Comment by jwyang** [comment]
Hi, @EldorIbragimov , it is highly dependent on your training batch size actually. So generally, if you trained with batchsize=1, then 1e-3 should be fine, otherwise, increase the learning rate a bit. In terms of the learning rate decay step, it should be 3-5 epochs for each decay in general.
